support-frontend/app/services/stepfunctions/SupportWorkersClient.scala

```diff
@@ -154,7 +154,7 @@ class SupportWorkersClient(
         user = user,
         giftRecipient = giftRecipient,
         product = request.body.product,
-        analyticsInfo = AnalyticsInfo(giftRecipient.isDefined, PaymentProvider.fromPaymentFields(request.body.paymentFields.left.toOption)),
+        analyticsInfo = AnalyticsInfo(giftRecipient.isDefined, PaymentProvider.fromPaymentFields(request.body.paymentFields.left.toOption), requestId),
```

Comment:
Why do we need to pass `requestId` into `AnalyticsInfo` now? Isn't this unnecessary duplication as we have it at the top level as well?

Reply:
just because I didn't want to go back and alter the other places that use it.  Although it may not be many places in reality - I will check.

Reply:
I think the primary reason for this might have gone away - I will have a look.

Reply:
removed, it was totally unnecessary in the end